week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -52,20 +50,20 @@ function sortArray() {
 
 function first5() {
   let numbers = [1, 2, 3, 4, 5, 6, 7, 8];
-  let sliced;
-
+  let sliced = numbers.splice(0, 5);
+  // console.log(sliced);
```

Comment:
this is a bit different. While your solution stands correctly
A better way is to use Slice method (give it a search)